docs/docs/create-source-plugin.md

```diff
@@ -53,7 +53,7 @@ What does the code look like?
 A source plugin is a normal NPM package. It has a package.json with optional
 dependencies as well as a `gatsby-node.js` where you implement Gatsby's Node.js
 APIs. Gatsby supports node versions back to Node 4 and as it's common to want to
-use more modern Node.js and JavaScript syntax, many plugins write code in a
+use more modern Node.js and JavaScript syntax, many plugins write code in an
```

Comment:
```suggestion
use more modern Node.js and JavaScript syntax, many plugins write code in a
```

This is already correct. An only for words that begin with a vowel or _sound_ like they begin with a vowel.

Reply:
@yeskunall I just read an article about how the way things sound out loud does determine whether an "a" or "an" sounds right.

Another suggestion here is just to use the word "source" instead of "`src`" because it's clearer if someone is using a screen reader!